config/merge.go

```diff
@@ -20,56 +17,44 @@ var (
 	}
 )
 
-// MergeServices merges a compose file into an existing set of service configs
-func MergeServices(existingServices *Configs, environmentLookup EnvironmentLookup, resourceLookup ResourceLookup, file string, bytes []byte) (map[string]*ServiceConfig, error) {
-	configs := make(map[string]*ServiceConfig)
-
-	datas := make(RawServiceMap)
-	if err := yaml.Unmarshal(bytes, &datas); err != nil {
-		return nil, err
-	}
-
-	if err := Interpolate(environmentLookup, &datas); err != nil {
-		return nil, err
+// Merge merges a compose file into an existing set of service configs
+func Merge(existingServices *ServiceConfigs, environmentLookup EnvironmentLookup, resourceLookup ResourceLookup, file string, bytes []byte) (map[string]*ServiceConfig, map[string]*VolumeConfig, map[string]*NetworkConfig, error) {
```

Comment:
I think we could make this a struct (the return part), something like `ProjectConfig` :angel:.
But I think that could be done in a follow-up PR :+1: